clang/lib/Driver/ToolChains/Arch/RISCV.cpp

```diff
@@ -42,9 +42,10 @@ static bool getArchFeatures(const Driver &D, StringRef Arch,
     return false;
   }
 
-  (*ISAInfo)->toFeatures(
-      Features, [&Args](const Twine &Str) { return Args.MakeArgString(Str); },
-      /*AddAllExtensions=*/true);
+  const auto ISAInfoFeatures = (*ISAInfo)->toFeatures(/*AddAllExtension=*/true,
+                                                      /*IgnoreUnknown=*/false);
+  Features.insert(Features.end(), ISAInfoFeatures.begin(),
+                  ISAInfoFeatures.end());
```

Comment:
```suggestion
  llvm::append_range(Features, (*ISAInfo)->toFeatures(/*AddAllExtension=*/true, /*IgnoreUnknown=*/false));
```

Reply:
Argh I was trying to be too clever and forgot that we need to call Args.MakeArgString, I've just replaced it with the push_back loop again.